from .vacancy_pb2 import CreateVacancyRequest
from .vacancy_pb2 import VacancyResponse
from .vacancy_pb2 import UpdateVacancyRequest
from .vacancy_pb2 import VacancyRequest
from .vacancy_pb2 import DeleteVacancyResponse
from .vacancy_pb2 import GetVacanciesRequest
from .vacancy_pb2 import Vacancy
from .vacancy_pb2_grpc import VacancyServiceStub
